### ISSUE TITLE
Scatter plot hangs when using color var.

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#1581]
In parallel if there are more processors than chunks to process - a common case when material selection is used.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Cyrus Harrison
Original creation: 08/17/2010 02:50 pm
Original update: 08/17/2010 05:23 pm
Ticket number: 348
